examples/with-jsx/next.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path')
+const nextPagesDir = path.join(__dirname, 'pages')
+
+const babelLoader = {
+  loader: 'babel-loader',
+  options: {}
+}
+
+const emitLoader = {
+  loader: 'emit-file-loader',
+  options: {
+    name: 'dist/[path][name].js'
+  }
+}
+
+const jsxLoader = {
+  test: /\.jsx$/,
+  use: [
+    emitLoader,
+    babelLoader
+  ],
+  exclude: /node_modules/,
+  include: [
+    __dirname,
+    nextPagesDir
+  ]
+}
+
+module.exports = {
+  // Add extensions
+  pagesExtensions: [
```

Comment:
Would it be more convenient for users to be able to specify extensions as `.${ext}`?

- https://webpack.js.org/configuration/resolve/#resolve-extensions
- https://eslint.org/docs/user-guide/command-line-interface#--ext